kotlinx-coroutines-debug/build.gradle

```diff
@@ -20,6 +20,10 @@ configurations {
 
 dependencies {
     compileOnly "junit:junit:$junit_version"
+    compileOnly "org.junit.jupiter:junit-jupiter-api:$junit5_version"
+    testCompile "org.junit.jupiter:junit-jupiter-api:$junit5_version"
```

Comment:
should `testCompile` duplicate `compileOnly`?

Reply:
Not at this point: other dependencies in `testCompile` depend on `junit-jupiter-api` transitively. I removed the duplicate.